templates/flutter/lib/src/client_offline_mixin.dart.twig

```diff
@@ -183,7 +183,7 @@ class ClientOfflineMixin {
 
   Future<void> checkOnlineStatus() async {
     try {
-      final url = Uri.parse('{{spec.endpoint}}/version');
+      final url = Uri.parse('{{sdk.url}}/version');
```

Comment:
but there is this `/version` ? will this work?

Reply:
Yep, https://appwrite.io/version works